index.js

```diff
@@ -117,6 +121,39 @@ f1.prototype = {
    */
   targets: function( targets ) {
 
+    var instance = this;
+
+    if (targets) {
+      _.forEach(targets, function (el, elementId) {
+        var elementData = {
+          id: elementId,
+          get states() {
+            return _.keys(instance.defStates).reduce(function (dict, key) {
+              dict[key] = instance.defStates[key][elementId];
+              return dict;
+            }, {});
+          },
+          set states(newStates) {
+            // TODO: what to do about state deletions? this only handles modifications or new states
+            Object.keys(newStates).forEach(function (stateId) {
+              instance.defStates[stateId][elementId] = newStates[stateId];
+            });
+            instance.go(instance.state);
+          },
+          get transitions() {
+            // TODO. Note: by default it's in the form of [{animation: {elementId1: { properties }, anotherElement: { properties }}, from, to}]
+            // should change it to [{animation: { properties }, from, to}]
+            return [];
+          },
+          $go: instance.go.bind(instance)
+        };
+        // add to global so the devtool can access the list of elements
+        global.f1targets[elementId] = elementData;
```

Comment:
Maybe ES6 Symbol or WeakMap could be used here instead since they often solve issues like this.

`global` anything should probably not be included in a module like `f1`, though. Almost always, mutating global state in modules is a recipe for disaster.